test/Microsoft.IdentityModel.Tokens.Tests/Json/JsonWebKeySetTheoryData.cs

```diff
@@ -14,7 +14,7 @@ public class JsonWebKeySetTheoryData : TheoryDataBase
 
         public IList<SecurityKey> ExpectedSigningKeys { get; set; }
 
-        public string Json { get; set; }
+        public string Json { get; set; } = string.Empty;
```

Comment:
```suggestion
        public string Json { get; set; }
```

We don't need to init properties to the default values.